resources/leiningen/new/kraken_works/script/build

```diff
@@ -65,4 +71,6 @@ perl -p -i -e "s/^Environment=CONTAINER=.*$/Environment=CONTAINER=${SERVICE}/" $
 if hash buildkite-agent 2>/dev/null ; then
   echo '--- saving service file'
   buildkite-agent artifact upload ${SERVICE}@.service
```

Comment:
We can get rid of the previous two lines as well! 🔪

Reply:
:hocho: :+1:

Reply:
Actually looks like a bunch of service template stuff can get removed... another commit incoming.